highfidelity/hifi
# Reselecting same entity in list will unselect it

Fix the same entity from being re-selectable multiple times when holding Ctrl and clicking on it in the entity list. When re-selecting the same entity it will be considered unselecting it instead.

Also fix indentations in entityList.js.

Fixes: https://highfidelity.manuscript.com/f/cases/17294/

Test Plan:
- Enter Interface in desktop mode
- Open Create and select an entity in the entity list
- Verify the entity row gets highlighted, the correct entity is selected in world, and that "1 entity selected appears" at the bottom of the list
- Hold Ctrl key and select the same entity in the entity list again
- Verify the entity gets unselected both in the list and in world, and the bottom text goes back to "X entities found"
- Hold Ctrl key and select 2+ entities in the entity list
- Verify the correct entities get selected in world, upon selecting the 2nd entity the edit handles change to multi-selection (corner scale cubes and scale spheres are no longer present and bounding box encompasses all selected entities), both entity rows are highlighted in the list, and "X entities selected" text appears for number selected
- While holding Ctrl key re-select a selected entity in the list
- Verify that the correct entity gets unselected both in the list and in world, and the bottom text's "entities selected" counter goes down by 1
